tests/distributed/test_pynccl.py

```diff
@@ -129,7 +131,7 @@ def worker_fn_with_cudagraph():
         # run something in the default stream to initialize torch engine
         a = torch.ones((4, 4), device=f'cuda:{comm.rank}')
         torch.cuda.synchronize()
-        with torch.cuda.graph(graph, stream=comm.stream):
+        with torch.cuda.graph(graph, stream=comm.stream), comm.enable():
```

Comment:
> Why do we need comm.enable here?

Because pynccl is disabled in eager mode (the default mode).

> Also, IIRC, python 3.8 does not support multiple with statements in a single line?

The code works fine in python 3.8:

```python
from contextlib import nullcontext

with nullcontext(), nullcontext():
    print("Hello, world!")
```